src/pages/pelico-profil.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+
+import { Box } from '@mui/material';
+
+import { Base } from 'src/components/Base';
+import { PelicoProfilNavigation } from 'src/components/accueil/PelicoProfilNavigation';
+import { UserContext } from 'src/contexts/userContext';
+import { primaryColor } from 'src/styles/variables.const';
+
+//
+const PelicoProfil = () => {
+  const { user } = useContext(UserContext);
+
+  return (
+    <>
+      <Base rightNav={<PelicoProfilNavigation activeUser={user} />}>
```

Comment:
To avoid the null check error : 
`<>
      {user && (
        <Base rightNav={<PelicoProfilNavigation activeUser={user} />}>....`